app/views/proposals/details/_top_notification.html.haml

```diff
@@ -0,0 +1,6 @@
+.medium-12.top-notification-wrapper
+  .row
+    .medium-12.column
+      = image_tag 'icon-configuration.png'
+      %p
+        This is a beta version of the Request Details page. #{link_to 'Switch to previous design', proposals_revert_design_path} if you prefer.
```

Comment:
instead of a route that deletes a cookie and redirects, did you consider deleting the cookie client-side via JS and then reloading the same page?